go.mod

```diff
@@ -573,7 +571,16 @@ require (
 	modernc.org/sqlite v1.35.0 // indirect
 	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.31.0 // indirect
 	sigs.k8s.io/json v0.0.0-20241010143419-9aa6b5e7a4b3 // indirect
-	sigs.k8s.io/randfill v1.0.0 // indirect
+)
+
+require (
+	github.com/grafana/grafana/pkg/build v0.0.0-20250408211749-365bd2b2089c
```

Comment:
This should not be included in the imports, did you link Enterprise while generating the file?

Reply:
possibly,will revert! thanks for catching it